packages/klighd-core/src/proxy-view/proxy-view.tsx

```diff
@@ -476,6 +465,13 @@ export class ProxyView extends AbstractUIExtension {
 
     //////// General helper methods ////////
 
+    /** Checks if `b1` is (partially) in `b2`. */
```

Comment:
I think the decoupling from the depth map is good. Especially since what you used from it was only this very specific functionality. I also think that this `isInBounds` method is more general than any specific feature and should reside in a more general place. There are a few different -util files, maybe there is something suitable there already.

Reply:
Good idea, I was hoping in the foreseeable futurue we could look at the methods together to see which ones could be moved to general util-functions (and which util-file to add them to)